### PR TITLE
Improve readability and amend some typos of APPENDIX in authentication.md

### PR DESCRIPTION
**Pull request by hw-qiaolei**
Improve readability and amend some typos of APPENDIX in authentication.md

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -123,14 +123,14 @@ into apiserver start parameters.
     `tar xzf easy-rsa.tar.gz`
     `cd easy-rsa-master/easyrsa3`
     `./easyrsa init-pki`
-1.  Generate a CA. (--batch set automatic mode. --req-cn default CN to use.)
-    `./easyrsa --batch "--req-cn=${MASTER_IP}@date +%s" build-ca nopass`
-1.  Generate server certificate and key.
+2.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default CN to use.)
+    ``./easyrsa --batch "--req-cn=${MASTER_IP}@`date +%s`" build-ca nopass``
```
Comment (erictune) [inline docs/admin/authentication.md]: I know about single backtick and triple backtick, but I haven't seen double backtick in markdown.  What is that for?

File: docs/admin/authentication.md [hunk 2]
```diff
@@ -123,14 +123,14 @@ into apiserver start parameters.
     `tar xzf easy-rsa.tar.gz`
     `cd easy-rsa-master/easyrsa3`
     `./easyrsa init-pki`
-1.  Generate a CA. (--batch set automatic mode. --req-cn default CN to use.)
-    `./easyrsa --batch "--req-cn=${MASTER_IP}@date +%s" build-ca nopass`
-1.  Generate server certificate and key.
+2.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default CN to use.)
```
Comment (erictune) [inline docs/admin/authentication.md]: The numbered list is still not formatting right.  You might need a second line break before the first item.

File: docs/admin/authentication.md (hunk 1 above)
Comment (hw-qiaolei) [inline docs/admin/authentication.md]: two backticks make ``"@`date +%s`"`` displays corretly, while single backtricks not(`"@`date +%s`"` ). You can test it.

File: docs/admin/authentication.md (hunk 2 above)
Comment (hw-qiaolei) [inline docs/admin/authentication.md]: Done. Please check, thx.

**Pull request closed without merging**